character.cpp

```diff
@@ -2217,6 +2217,7 @@ int chara_relocate(int prm_784, int prm_785, int prm_786)
         cdata[tc_at_m125].relationship = relationship;
         cdata[tc_at_m125].original_relationship = original_relationship;
         cdata[tc_at_m125].hate = hate;
+        cdata[tc_at_m125].enemy_id = enemy_id;
```

Comment:
Indeed it is related to this pull request, "enemy_id is unused" warning, but I think it is better to separate into another issue/pull request because it is not just refactoring, but a bug fix.

Reply:
Ok, will do.